woocommerce/woocommerce-gateway-stripe
# Adds comprehensive test coverage for UPE checkouts

# Changes proposed in this Pull Request:

Issue: Link to the GitHub issue this PR addresses (if appropriate).
Fixes #2016 

This PR adds more comprehensive test coverage for all regular flows during UPE checkouts (excluding any that are subscription related or checkouts using a saved payment method--which are already comprehensively covered).


# Testing instructions

Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?

1. Pull the changes in this PR and start up your local environment (`npm run up`).
2. Run the tests (`npm run test:php`).
3. Hopefully, see green dots all across the board.
4. Celebrate!

-------------------
